ext/geoip2/rb_compat.h

```diff
@@ -0,0 +1,12 @@
+#ifndef __RB_COMPAT_H__
+#define __RB_COMPAT_H__
+
+#include <ruby.h>
+
+#ifndef HAVE_RB_SYM2STR
+#  define SYM2STR(name) (rb_id2str(SYM2ID(name)))
+#else
+#  define SYM2STR(name) (rb_sym2str(name))
+#endif
+
```

Comment:
How about following code?
We can just remove rb_compat.h when we drop old version of Ruby support.

```c
#ifndef HAVE_RB_SYM2STR
#  define rb_sym2str(name) (rb_id2str(SYM2ID(name)))
#endif
```